### DISCUSSION TITLE
Remove password for Google linked account

### DISCUSSION THREAD

**Discussion by vilgotf** [monkeytypegame/monkeytype#4604] [Ideas]
Accounts originally created with an email and password but are then linked to a Google account retain their email + password sign in option. While having two sign in options can certainly be useful, it'd also be nice to be able to remove one's, redundant, password.

**Comment by Miodec** [comment]
Ive had many users message support because they have lost access to their gmail account. Password auth is there as a backup, so if its enabled, it can't be removed.
